src/layouts/components/Homepage/AnnouncementBody.tsx

```diff
@@ -0,0 +1,240 @@
+import { Text, Box, FormControl } from "@chakra-ui/react"
+import { DragDropContext } from "@hello-pangea/dnd"
+import {
+  FormLabel,
+  Input,
+  FormErrorMessage,
+  Button,
+  DatePicker,
+  Textarea,
+} from "@opengovsg/design-system-react"
+import _ from "lodash"
+import moment from "moment"
+import { BiPlus } from "react-icons/bi"
+
+import { useEditableContext } from "contexts/EditableContext"
+
+import { Editable } from "layouts/components/Editable"
+
+import { AnnouncementError, AnnouncementOption } from "types/homepage"
+
+const MAX_ANNOUNCEMENTS = 5
+
+interface AnnouncementBodyProps {
+  errors: {
+    announcementItems: AnnouncementError[]
+  }
+  announcementItems: Partial<AnnouncementOption>[]
+}
+
+const toTemplateDateFormat = (date?: string) => {
+  const formattedDate = moment(date, "DD/MM/YYYY", true).format("DD MMMM YYYY")
+  if (formattedDate === "Invalid date") {
+    return date
+  }
+  return formattedDate
+}
+
+const toCmsPanelDateFormat = (date?: string) => {
+  const formattedDate = moment(date, "DD MMMM YYYY", true).format("DD/MM/YYYY")
+  if (formattedDate === "Invalid date") {
+    return date
+  }
+  return formattedDate
+}
```

Comment:
nit: quite hard to tell apart at first glance, can just add a comment on either one stating the difference so ppl don't edit/delete accidentally

Reply:
c9816a6d18975d38286812cac138cd43c341f91b